sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -48,5 +55,29 @@ pip install azure-ai-textanalytics --pre
 
 ## Next steps
 
-Check out the [API reference documentation](https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics-v3-0-Preview-1/operations/Languages) to learn more about
+Check out the [API reference documentation](https://aka.ms/azsdk-python-textanalytics-ref-docs) to learn more about
 what you can do with the Azure Text Analytics client library.
+
+|**Advanced Sample File Name**|**Description**|
```

Comment:
Can we move these samples up to the main table? These are still important scenarios

Reply:
I was wondering how much of an advanced use case this is. I talked to @johanste yesterday and he said it was an advanced user case, and following the [sample guidance](https://microsoft.sharepoint.com/teams/AzureDeveloperExperience/_layouts/15/Doc.aspx?sourcedoc={33bd5249-4696-4382-b9f5-c5c1436a2b36}&action=edit&wd=target%28Vanadium%20Planning%20and%20Status.one%7Cf621d863-c3ec-4e47-9e60-08ed695aa5ab%2FSamples%20Guidance%7C2daef4da-3fd3-4fce-834d-d7f98ef72dc4%2F%29) it said to include advanced samples under next steps. Definitely open for discussion though and will defer to you because you know more about how likely this scenario will happen than I do

Reply:
The getting of the statistics/model_version/raw resp in the response hook could be considered advanced, but this sample also demonstrates how to use alternative input that isn't just `list[str]`. I'm wondering if we keep this in the "advanced" section maybe we should have separate sample that demonstrates alt input?

Reply:
I agree, I'll split it up and add another sample titled sample_alternative_document_input.py (or something like that) and pass input alternatively to one of the endpoints. thanks!

Reply:
Sounds good, thank you!